manifests/init.pp

```diff
@@ -6,6 +6,8 @@
   Hash $forward,
   Hash $stub,
   Hash $record,
+  Optional[Unbound::Local_zone] $local_zone,
```

Comment:
Just looking over the man page, looks like multiple local-zone definitions are allowed.  This should probably be `Optional[Array[Unbound::LocalZone]]` to allow multiple local zones.  The template will also need adjusting.

Reply:
`type Unbound::Local_zone = Hash[String, Unbound::Local_zone_type]` so you can pass multiple zones here, template already accounts for this

Reply:
Yep, I just missed that.